dvc/scm.py

```diff
@@ -123,3 +125,56 @@ def resolve_rev(scm: "Git", rev: str) -> str:
             if len(ref_infos) > 1:
                 raise RevError(f"ambiguous Git revision '{rev}'")
         raise RevError(str(exc))
+
+
+def iter_revs(
```

Comment:
`iter_revs` is no longer an iterator now.

Reply:
Any name to suggest?

Reply:
We can modify these in #7245